### DISCUSSION TITLE
bookmarks whether to accept PR？

### DISCUSSION THREAD

**Discussion by imyuanx** [alefragnani/vscode-bookmarks#509] [Q&A] [answered]
I hope show `Clear  From All Files` in side bar, I have some code give bookmarks, But i not have `bookmarks-core` so I can't test and i don't know bookmarks whether to accept PR, So i'll submit it first but it's up to you.
Thanks, Great job.

 [PR: https://github.com/alefragnani/vscode-bookmarks/pull/508](https://github.com/alefragnani/vscode-bookmarks/pull/508)

**Comment by alefragnani** [comment, marked as answer]
Hi @yunying1 ,

Thanks, but in most cases no, the extension is not accepting PRs. Basically because the `vscode-bookmarks-core` repository is private. The most _common/possible_ case for PRs should be translations (`package.nls.xx.json`) because you can test it updating the installed extension.

Additionally, PRs should refers a previously open/discussed issue, specially if you are adding a new feature. Your PR, has no issue linked, so I ask you to open an issue first, to discuss and gather feedback from other users.

Hope this helps

**Reply by imyuanx** [reply]
Thanks, i know now.

I'll open a issue, because i really need this feature!
